presto-docs/src/main/sphinx/plugin/redis-hbo-provider.rst

```diff
@@ -2,14 +2,16 @@
 Redis HBO Provider
 =======================
 
-Redis HBO Provider allows loading a custom configured Redis Client for storing/retrieving Historical Stats. The Redis client is stateful and is based on
-`Lettuce <https://github.com/lettuce-io/lettuce-core>`_ . Both RedisClient and RedisClusterClient are supported, RedisClusterAsyncCommandsFactory is meant to be extended by the user for custom configurations
+Redis HBO Provider supports loading a custom configured Redis Client for storing and retrieving historical stats for Historical Based Optimization (HBO). The Redis client is stateful and is based on
+`Lettuce <https://github.com/lettuce-io/lettuce-core>`_. Both RedisClient and RedisClusterClient are supported, RedisClusterAsyncCommandsFactory is meant to be extended by the user for custom configurations.
 
 
 Configuration
 -------------
 
-Create ``etc/catalog/redis-provider.properties`` to mount the Redis HBO Provider Plugin, replacing the properties as appropriate:
+Create ``etc/catalog/redis-provider.properties`` to mount the Redis HBO Provider Plugin. 
+
+Edit the configuration properties as appropriate:
```

Comment:
Are there default values for these configuration properties that could be added to this table?

Reply:
I have added some defaults, most of the other config properties will need user enviroment specific changes

Reply:
Actually we can add a default column

Reply:
Thanks! I added a default column in the second table - the four properties in "Coordinator Configuration for Historical Based Optimization", as you provided default values for all of them. Here's a screenshot of my local build showing the Default Value column added to that table. 

<img width="1056" alt="Screenshot 2024-02-01 at 9 49 14 AM" src="https://github.com/prestodb/presto/assets/7013443/8707876f-d275-4827-bac4-c65ee1600b64">

Reply:
Are there default values that could be documented for any of the eight properties in the first table (``coordinator`` through ``hbo.redis-provider.cluster-mode-enabled``)?

Reply:
Actually, we don't have a default value set as of now. I will try to add default values in a PR soon. 
Would it work if we added a sample property column?

Reply:
I have added some examples below in the documentation though

Reply:
> Actually, we don't have a default value set as of now. I will try to add default values in a PR soon. Would it work if we added a sample property column?

I see the examples in the codeblock, those are a big help for the user. 

Docs should describe the current state of the software - if the default values for those don't exist then we don't need to document them. When you add default values in a future PR, we can document those values at that time by updating this page as part of that PR. Does that seem reasonable?